temperature-lcd/maven/src/main/java/iotdk/example/DisplayTemperatureOnLCD.java

```diff
@@ -130,10 +130,11 @@ static void temperature_update(GroveTemp temp, GroveButton button, GroveLed led,
     }
     // Set true if using a Grove Pi Shield, else false
     static final boolean USING_GROVE_PI_SHIELD = true;
-    static String unknownPlatformMessage = "This sample uses the MRAA/UPM library for I/O access, " +
-        "you are running it on an unrecognized platform. " +
-        "You may need to modify the MRAA/UPM initialization code to " +
-        "ensure it works properly on your platform.\n\n";
+	
+	static int dInPin = 4;
+	static int dOutPin = 3;
+	static int aPin = 0;
+	static int i2cPort = 0;
```

Comment:
Unavailable on UP2, see previous suggestions or throw error when not using GrovePi+ shield like in C++ samples.

Reply:
Made it the same way as cpp-file.